src/checkout/CheckoutForm.tsx

```diff
@@ -118,21 +117,6 @@ const CheckoutForm: FC = () => {
                 </SafeBlankLink>
                 .
               </p>
-
-              {isPaygCreditActive && (
-                <GoogleOptimizeExperiment
-                  experimentID={CREDIT_250_EXPERIMENT_ID}
-                  variants={[
-                    <div
-                      className="checkout-form--banner"
-                      key="checkout-form-banner"
-                    >
-                      <strong className="checkout-banner--credit">$250</strong>
-                      <p>credit applied</p>
-                    </div>,
-                  ]}
-                />
-              )}
```

Comment:
This was creating the "sash" which is from an old experiment. Per the specs and requirements, use a banner instead.

Not this:
<img width="1728" alt="Screen Shot 2022-05-20 at 9 47 49 AM" src="https://user-images.githubusercontent.com/10736577/169603057-3bfb7eff-2433-450c-8e36-c60b37e4333e.png">


But this:
<img width="1728" alt="Screen Shot 2022-05-20 at 10 17 47 AM" src="https://user-images.githubusercontent.com/10736577/169603175-c8263c11-fcec-4250-8545-1be845da6988.png">